Author: Jürgen Salk <jsa@debian.org>
Description: The original maintainer Jürgen Salk applied
 a set of patches to the original code.  This file contains
 changes to C++ code
Forwarded: no

--- a/dcmqrdb/etc/dcmqrscp.cfg
+++ b/dcmqrdb/etc/dcmqrscp.cfg
@@ -21,8 +21,8 @@
 # SpecificCharacterSet = "ISO_IR 192", override, discard, transliterate
 
 #
-# UserName      = <not used>
-# GroupName     = <not used>
+UserName      = dcmtk
+GroupName     = dcmtk
 
 HostTable BEGIN
 #
@@ -38,12 +38,13 @@
 # NOTE: in the current implementation you cannot substitute an IP address
 # for a hostname.
 #
-acme1           = (ACME1, acmehost1, 5678)
-acme2           = (ACME2, acmehost2, 5678)
-acmeCTcompany   = acme1, acme2
-united1         = (UNITED1, unitedhost1, 104)
-united2         = (UNITED2, unitedhost2, 104)
-unitedMRcompany = united1, united2
+# Example:
+#acme1           = (ACME1, acmehost1, 5678)
+#acme2           = (ACME2, acmehost2, 5678)
+#acmeCTcompany   = acme1, acme2
+#united1         = (UNITED1, unitedhost1, 104)
+#united2         = (UNITED2, unitedhost2, 104)
+#unitedMRcompany = united1, united2
 #
 HostTable END
 
@@ -61,8 +62,9 @@
 #       VendorName = SymbolicName
 # The symbolic name should be defined in the HostTable.
 #
-"Acme CT Company"   = acmeCTcompany
-"United MR Company" = unitedMRcompany
+# Example:
+#"Acme CT Company"   = acmeCTcompany
+#"United MR Company" = unitedMRcompany
 #
 VendorTable END
 
@@ -80,8 +82,13 @@
 #               Entry in HostTable                      |
 #               ANY
 #
-COMMON       /home/dicom/db/COMMON       R  (200, 1024mb) ANY
-ACME_STORE   /home/dicom/db/ACME_STORE   RW (9, 1024mb)   acmeCTcompany
-UNITED_STORE /home/dicom/db/UNITED_STORE RW (9, 1024mb)   unitedMRcompany
+# Example:
+#
+#ACME_STORE   /var/lib/dcmtk/db/ACME_STORE   RW (9, 1024mb)   acmeCTcompany
+#UNITED_STORE /var/lib/dcmtk/db/UNITED_STORE RW (9, 1024mb)   unitedMRcompany
+#
+# Uncomment and adjust the following lines for a common r/rw storage area:
+#READWRITE     /var/lib/dcmtk/db/READWRITE    RW (10, 1024mb)  ANY
+#READ          /var/lib/dcmtk/db/READ         R  (200, 1024mb) ANY
 #
 AETable END
--- a/dcmqrdb/docs/dcmqrcnf.txt
+++ b/dcmqrdb/docs/dcmqrcnf.txt
@@ -46,8 +46,8 @@
 MaxPDUSize           = 8192
 MaxAssociations      = 20
 SpecificCharacterSet = fallback
-UserName             = (do not change user)
-GroupName            = (do not change group)
+UserName             = dcmtk
+GroupName            = dcmtk
 
 Available options for specific character sets are:
 
